### PR TITLE
Michael Kudlaty Deliverable 4 - JavaLife

### PR DESCRIPTION
**Pull request by kuds**
Pull request includes the summary, all necessary java files and the file containing the unit test.

### CODE REVIEW

File: com/laboon/World.java [hunk 1]
```diff
@@ -114,7 +143,8 @@ public World iterate() {
 	 * @return String representation of world
 	 */
 	
-	public String toString() {
+        //Oringal Method
```
Comment (laboon) [inline com/laboon/World.java:146]: "Original"

File: test/toStringTests.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    }    
+      
+    //Creates a new world with a 1 by 1 grid, checks to see if the original and
+    //modified toString methods are equal. 
+    @Test
+    public void checkingMap1()
+    {
+        World world = new World (1, 5, 30);
+        
+        assertEquals(world.toString(), world.toStringOriginal());
```
Comment (laboon) [inline test/toStringTests.java:61]: These tests will be obsolete as soon as the original toString() is removed, which should be soon if we don't need it anymore

File: test/toStringTests.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        
+        for(int i = 0; i < 1000; i++)
+        {
+            world = world.iterate();
+            assertEquals(world.toString(), world.toStringOriginal());  
+        }
+    }        
+
+    
+}
```
Comment (laboon) [inline test/toStringTests.java:133]: Edge cases?
